lib/hawkular/version.rb

```diff
@@ -4,5 +4,5 @@
 # @see https://github.com/hawkular
 module Hawkular
   # Version of the Hawkular Ruby Gem
-  VERSION = '2.2.1'
+  VERSION = '2.3.0'.freeze
```

Comment:
On 26 Jul 2016, at 13:24, abonas wrote:

>> @@ -4,5 +4,5 @@
>>  # @see https://github.com/hawkular
>>  module Hawkular
>>    # Version of the Hawkular Ruby Gem
>> -  VERSION = '2.2.1'
>> +  VERSION = '2.3.0'.freeze
>
> why does it need a freeze?

I think Rubocop suggested it. Isn't it good practice
to freeze the "constants"?
But then idc